app/controllers/chargeback_assignment_controller.rb

```diff
@@ -31,12 +31,13 @@ def change_tab
   def form_field_changed
     return unless load_edit("cbassign_edit__#{params[:id]}", "index")
     get_form_vars
+    build_tabs
     render :update do |page|
       page << javascript_prologue
       except = %i[cbshow_typ cbtag_cat cblabel_key]
       changed = (@edit[:new].except(*except) != @edit[:current].except(*except))
       prefix = @edit[:new][:type].downcase
-      page.replace("#{prefix}_assignments_div", :partial => "#{prefix}_assignments") if params[:cbshow_typ] || params[:cbtag_cat] || params[:cblabel_key]
+      page.replace(@edit[:new][:type], :partial => "#{prefix}_assignments") if params[:cbshow_typ] || params[:cbtag_cat] || params[:cblabel_key]
```

Comment:
@kavyanekkalapu there is no need to set `prefix` variable, you can just directly use `@edit[:new][:type].downcase` instead of prefix variable here.